pycon-au-2010/videos/pyconau-2010--using-python-in-a-scientific-real-t.json

```diff
@@ -19,12 +19,12 @@
     "pyconau2010",
     "science"
   ],
-  "thumbnail_url": "http://a.images.blip.tv/Pyconau-UsingPythonInAScientificRealtimeDataCollectionNetwork865.png",
+  "thumbnail_url": "https://ia801202.us.archive.org/27/items/pyvideo_463___pyconau-2010-using-python-in-a-scientific-real-time-data-collection-network/pyvideo_463___pyconau-2010-using-python-in-a-scientific-real-time-data-collection-network.thumbs%2F463_pyconau-2010-using-python-in-a-scientific-real-time-data-collection-network_000114.jpg",
   "title": "PyConAU 2010: Using Python in a scientific real-time data collection network",
   "videos": [
     {
       "type": "flv",
-      "url": "http://05d2db1380b6504cc981-8cbed8cf7e3a131cd8f1c3e383d10041.r93.cf2.rackcdn.com/pycon-au-2010/463_pyconau-2010-using-python-in-a-scientific-real-time-data-collection-network.flv"
+      "url": "https://archive.org/details/pyvideo_463___pyconau-2010-using-python-in-a-scientific-real-time-data-collection-network"
```

Comment:
shouldn't this URL pour to the `.flv` directly?
There are also OGG and H.264 versions that are a bit slimmer

Reply:
A large number of our other videos link to Archive.org in the same way (sans  a direct link to a video file). I'm happy to change if you feel strongly about it though.